.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```

Comment:
These .idea files should not be a part of your PR. Please add .idea/ to .gitignore and remove these files